plugins/home/src/components/CustomHomepage/CustomHomepageGrid.tsx

```diff
@@ -207,9 +209,11 @@ export const CustomHomepageGrid = (props: CustomHomepageGridProps) => {
     [props],
   );
 
-  const defaultLayout = props.config
-    ? convertConfigToDefaultWidgets(props.config, availableWidgets)
-    : [];
+  defaultLayout =
```

Comment:
I'm not sure this is the right fix here. Should this be wrapped up in a `useMemo` similar to how `widgets` are done above?

Reply:
`useMemo ` also works